src/common/dcsvg.cpp

```diff
@@ -743,7 +973,7 @@ void wxSVGFileDCImpl::NewGraphicsIfNeeded()
 
     m_graphics_changed = false;
 
-    write(wxS("</g>\n"));
+    write(wxT("</g>\n"));
```

Comment:
Why? We do have an awful mess of `wxT()` and `wxS()` here and I don't say we should necessarily clean it up immediately, but at least let's not change correct latter ones into less correct former ones. And `wxS()` is better because `wxString` can be constructed from it more efficiently.